### PR TITLE
Folder

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#174 
#### What does this implement/fix? Explain your changes.
- `Folder` parameter added to `Static_Analysis` function
- `Folder` parameter added to `Dynamic_Analysis` function
- Document modified

### CODE REVIEW

File: opem/Functions.py [hunk 1]
```diff
@@ -287,23 +287,30 @@ def Output_Save(
...
         spliter = "\r\n"
     Art = text2art("Opem")
-    if Title not in os.listdir(os.getcwd()):
-        os.mkdir(Title)
-    opem_file = open(os.path.join(Title, Name + ".opem"), "w")
+    if not os.path.exists(Folder):
+        os.mkdir(Folder)
+    sim_folder = os.path.join(Folder, Title)
+    if not os.path.exists(sim_folder):
+        os.mkdir(sim_folder)
```
Comment (sadrasabouri) [inline opem/Functions.py:312]: What is this for?

File: opem/Functions.py [hunk 2]
```diff
@@ -318,19 +325,28 @@ def Output_Init(InputDict, Title, Name):
...
     :return: file object
     """
-    if Title not in os.listdir(os.getcwd()):
-        os.mkdir(Title)
-    csv_file = open(os.path.join(Title, Name + ".csv"), "w")
+    if not os.path.exists(Folder):
+        os.mkdir(Folder)
+    sim_folder = os.path.join(Folder, Title)
+    if not os.path.exists(sim_folder):
+        os.mkdir(sim_folder)
```
Comment (sadrasabouri) [inline opem/Functions.py:348]: similar here.

File: opem/Functions.py [hunk 3]
```diff
@@ -353,21 +369,26 @@ def None_Omit(Input_Str):
...
+    :type Folder: str
     :return: HTML file as file obj
     """
-    if Title not in os.listdir(os.getcwd()):
-        os.mkdir(Title)
+    if not os.path.exists(Folder):
+        os.mkdir(Folder)
+    sim_folder = os.path.join(Folder, Title)
+    if not os.path.exists(sim_folder):
+        os.mkdir(sim_folder)
```
Comment (sadrasabouri) [inline opem/Functions.py:388]: and here.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]

File: opem/Functions.py (hunk 3 above)
Comment (sepandhaghighi) [reply opem/Functions.py]: fixed in https://github.com/ECSIM/opem/pull/199/commits/4745d34ee7569d541ff815ae9ca334291fcb18fc

File: opem/Functions.py (hunk 2 above)
Comment (sepandhaghighi) [reply opem/Functions.py]: fixed in https://github.com/ECSIM/opem/pull/199/commits/4745d34ee7569d541ff815ae9ca334291fcb18fc

File: opem/Functions.py (hunk 1 above)
Comment (sepandhaghighi) [reply opem/Functions.py:312]: fixed in https://github.com/ECSIM/opem/pull/199/commits/4745d34ee7569d541ff815ae9ca334291fcb18fc

File: opem/Functions.py [hunk 4]
```diff
@@ -287,23 +287,30 @@ def Output_Save(
...
     """
     spliter = "\n"
     if 'win' not in sys.platform:
         spliter = "\r\n"
     Art = text2art("Opem")
-    if Title not in os.listdir(os.getcwd()):
-        os.mkdir(Title)
-    opem_file = open(os.path.join(Title, Name + ".opem"), "w")
+    if not os.path.exists(Folder):
+        os.mkdir(Folder)
```
[the remark this answers is not in the document]
Comment (sepandhaghighi) [reply opem/Functions.py]: fixed in https://github.com/ECSIM/opem/pull/199/commits/4745d34ee7569d541ff815ae9ca334291fcb18fc

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**